### PR TITLE
Nano S+ and X support

### PR DESCRIPTION
**Pull request by jonored**

### CODE REVIEW

File: .cargo/config [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
-[build]
-target = "thumbv6m-none-eabi"
+[target.nanos]
+rustflags = [
+  "-C", "passes=ledger-ropi",
+  "-C", "codegen-units=1",
+  "-C", "save-temps", # Fixes incremental builds by keeping .bc and .ll files
+  "-C", "embed-bitcode",
+  "--cfg", "target_os=\"nanos\"",
+  "--cfg", "target_device=\"nanosplus\"",
```
Comment (Ericson2314) [inline .cargo/config:26]: ?

File: .cargo/config [hunk 2]
```diff
@@ -1,24 +1,46 @@
...
+runner = "./speculos-wrapper -m nanos "
+
+[target.nanox]
+rustflags = [
+  "-C", "passes=ledger-ropi",
+  "-C", "codegen-units=1",
+  "-C", "save-temps", # Fixes incremental builds by keeping .bc and .ll files
+  "-C", "embed-bitcode",
+  "--cfg", "target_os=\"nanos\"",
+  "--cfg", "target_device=\"nanosplus\"",
```
Comment (Ericson2314) [inline .cargo/config:39]: ?

File: .cargo/config (hunk 1 above)
Comment (dfordivam) [reply .cargo/config:26]: This could be removed, this is from the previous commit I did, and nano_sdk branch used now does not make use of this.

Comment (Ericson2314) [reply .cargo/config:26]: Ok

**Comment by Ericson2314** [comment]
Has been done since with cleaned up history.

**Pull request closed without merging**